include/Lv2Worker.h

```diff
@@ -47,9 +47,10 @@ class Lv2Worker
 {
 public:
 	// CTOR/DTOR/feature access
-	Lv2Worker(const LV2_Worker_Interface* iface, Semaphore* common_work_lock, bool threaded);
+	Lv2Worker(Semaphore* common_work_lock, bool threaded);
```

Comment:
I would prefer ``common_work_lock`` to be ``commonWorkLock``. It fits better with our style.